coldfront/plugins/qumulo/views/allocation_view.py

```diff
@@ -65,6 +70,8 @@ def form_valid(
 
             absolute_path = f"/{prepend_val}/{storage_filesystem_path}"
         validate_filesystem_path_unique(absolute_path)
+        if billing_cycle == "prepaid" and prepaid_billing_date > date.today():
```

Comment:
@cdeanj2 These two lines are the implementation of a business rule. I recommended factoring out to a method. Move the method call after line 56, where the form data for the billing_cycle and prepaid_billing_date assignment.